pkg/controlplane/controller/leaderelection/run_with_leaderelection.go

```diff
@@ -29,12 +29,11 @@ import (
 	"k8s.io/klog/v2"
 )
 
-var (
-	// TODO: Eventually these should be configurable
-	LeaseDuration = 15 * time.Second
```

Comment:
These are also defined in a way that can be overridden in integration tests:
 https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/coordinatedleaderelection/leaderelection_test.go#L50-L54

I'd imagine these tests will fail at the moment, can you update the references to set these up properly?

Reply:
Yep, let me update those.